src/matlab2tikz.m

```diff
@@ -3514,14 +3514,7 @@ function warnFacetedInterp(m2t)
         cData = get(h, 'MarkerEdgeColor');
     end
 
-    matlabMarker = get(h, 'Marker');
-    markerFaceColor = get(h, 'MarkerFaceColor');
-    markerEdgeColor = get(h, 'MarkerEdgeColor');
-    hasFaceColor = ~isNone(markerFaceColor);
-    hasEdgeColor = ~isNone(markerEdgeColor);
-    markOptions = opts_new();
-    [tikzMarker, markOptions] = translateMarker(m2t, matlabMarker, ...
-        markOptions, hasFaceColor);
+    rawMarker = getMarkerStruct(m2t, h);
```

Comment:
Yeah, i couldnt think of a better one. Maybe we could go with markerStruct and add similar one for lineStruct, pathcStruct etc?